swagger.json

```diff
@@ -2723,6 +2723,71 @@
                     }
                 }
             }
+        },
+        "/contacts": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/contacts",
+                "description": "Fetch all contacts for the organization.",
+                "operationId": "GetOrganizationContacts",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of contacts.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/Contact"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/contacts/{contactId}": {
```

Comment:
Should this be `contact_id`?

Reply:
Not sure - for some entities (addresses, drivers) we camelcase the id and for others (assets, vehicles) we snake case it. @silverspace do you have opinions here?

Reply:
Let's `camelCase` JSON request/response bodies, and `spinal-case` query parameters and PATH parameters. https://paper.dropbox.com/doc/API-Best-Practices-HNfOvJK77p8UIC5BUFO8S

Reply:
Sorry I didn't fully read this question -- the name of the route template variable doesn't matter much to me, that's internal to our swagger spec.

Reply:
Let's use `snake_case` since that's what most of the rest of our swagger spec uses, just to be internally consistent.

Reply:
Changed to `contact_id`